src/libData/AccountData/AccountStoreSC.tpp

```diff
@@ -738,7 +738,14 @@ bool AccountStoreSC<MAP>::ParseCreateContractJsonOutput(
     receipt.AddError(NO_GAS_REMAINING_FOUND);
     return false;
   }
-  gasRemained = atoi(_json["gas_remaining"].asString().c_str());
+  try {
+    gasRemained =
+        boost::lexical_cast<uint64_t>(_json["gas_remaining"].asString());
```

Comment:
ensure when casting string to uint, it won't have any underflow/overflow issue if neccessary